src/Registration/Registration.jsx

```diff
@@ -61,11 +61,11 @@ export default function Registration() {
   };
 
   return (
-    <Container fluid="md">
-      <Card>
+    <>
```

Comment:
The component already has a parent component (<Card>). So there is no need for a fragment